tests/linearGaussian_vector_field_test.py

```diff
@@ -103,9 +104,9 @@ def test_c2st_vector_field_on_linearGaussian(
         posterior = inference.build_posterior(
             score_estimator,
             sample_with=method,
-            vectorfield_sampling_parameters={
-                "neural_ode_backend": "zuko",
-            },
+            posterior_parameters=VectorFieldPosteriorParameters(
+                neural_ode_backend="zuko"
```

Comment:
this is redundant because we only have `zuko`, we can remove it here and just use the defaults.